gui/src/components/widgets/line-chart.vue

```diff
@@ -239,7 +239,7 @@ export default class CortxLineChart extends Vue {
     if (document.hidden) {
       this.ispollThroughPut = false;
     } else {
-      this.initThrouthputStats(1800);
+      this.initThrouthputStats(this.prefetch?this.prefetch:1800);
```

Comment:
Formatting could have been better. There is no space between the control statements.

Reply:
I'll resolve this in the dashboard -performance PR.